cinema/models.py

```diff
@@ -27,6 +27,10 @@ class Actor(models.Model):
     first_name = models.CharField(max_length=255)
     last_name = models.CharField(max_length=255)
 
+    @property
+    def full_name(self):
+        return self.first_name + " " + self.last_name
```

Comment:
The `full_name` property in the `Actor` model is correctly implemented, concatenating the first and last names.